src/lib/i18next/resources.js

```diff
@@ -1,255 +1,265 @@
+      "Warning: changing the starting meter can cause consistency issues.": "သတိပေးချက် - စတင်မီတာကိုပြောင်းလဲခြင်းသည်ရှေ့နောက်ညီညွတ်မှုပြဿနာများဖြစ်စေနိုင်သည်။",
+      "Current [x]": "လက်ရှိ-",
+      "New [x]": "-အသစ်",
+      "Enter [x] (To Add)": "-ထည့်သွင်းရန်",
+      "Motivation": "အကြောင်းပြချက်",
+      "Warning: changing the total owed can lead to loss of revenue.": "သတိပေးချက် - စုစုပေါင်းကျသင့်ငွေကိုပြောင်းလဲခြင်းသည်ဝင်ငွေဆုံးရှုံးမှုကိုဖြစ်စေနိုင်သည်။",
+      "Why are you changing this?": "အကြောင်းပြချက်ရေးပါ...",
+      "Invoices": "ငွေတောင်းခံလွှာများ",
+      "Payments": "ငွေပေးချေမှု",
+      "After you click \"OK\", you will no longer be able to edit meter readings for this period.": "\"OK\"ကိုနှိပ်ပြီးနောက်သင်ဤကာလအတွက်မီတာဖတ်ခြင်းကိုပြောင်းနိုင်တော့မည်မဟုတ်ပါ။",
+      "You are about to create the invoice for this period.": "ကျေးဇူးပြုပြီးဒီလအတွက်ငွေတောင်းခံလွှာကိုအတည်ပြုပါ။ ",
+      "Calculated Usage": "အသုံးပြုမှုတွက်ချက်မီတာ",
+      "New Meter Reading (kWh)": "မီတာအသစ်ဖတ်ခြင်း (kWh)",
+      "After Edit [Payment]": "ပြောင်းပြီးနောက်",
+      "Current Amount [Payment]": "လက်ရှိပေးမည့်ငွေပမာဏ",
+      "Enter a new payment amount": "ငွေပေးချေမှုအသစ်ပမာဏပြောင်းခြင်း",
+      "New Amount [Payment]": "ငွေပေးချေမှုပမာဏအသစ်",
+      "After Payment": "ပေးချေပြီးနောက်ကျန်ငွေ",
+      "Enter a new payment": "ငွေပေးချေမှုအသစ်ရေးထည့်ခြင်း",
+      "New Payment": "ငွေပေးချေမှုအသစ်",
+      "Suggested Amount [Payment]": "ထပ်ပေးရန်လိုသောငွေပမာဏ",
+      "This customer has not been invoiced for 0 periods. Please add an amount to the invoice for these periods.": "ဤမီးအသုံးပြုသူအတွက်ငွေမတောင်းခံရသည်မှာ 0 ကြာပါပြီ။ ဒီကာလအတွက်ငွေတောင်းခံလွှာကိုရေးပါ။",
+      "Enter amount in Ks [Payment]": "အပိုငွေပေးချေမှုငွေပမာဏရေးပါ...",
+      "Ending Meter": "ယခုမီတာ",
+      "Period Usage": "ယူသုံးကာလ",
+      "Starting Meter": "စတင်မီတာ",
+      "Fixed [Tariff]": "ပုံသေစျေးနှုန်း",
+      "Free units [Tariff]": "အခမဲ့ယူနစ်",
+      "Per kWh [Tariff]": "တစ်ယူနစ်လျင်",
+      "Period Paid": "ပေးဆောင်ပြီးငွေ",
+      "To edit payments, click here or go to records below": "ငွေပေးချေမှုကိုပြောင်းပြင်ရန်ဤနေရာကိုနှိပ်ပါ(သို့မဟုတ်)အောက်တွင်ဖော်ပြထားသောစာရင်းသမိုင်းများကိုသွားပါ။",
+      "Amount Billed": "ကျသင့်ငွေ",
+      "Site": "ဆိုက်",
+      "History": "စာရင်းသမိုင်း",
+      "Go to settings to change this.": "၎င်းကိုပြောင်းလဲရန် Settings သို့သွားပါ။",
+      "This customer is deactivated.": "ဤမီးအသုံးပြုသူအားရပ်ဆိုင်းထားသည်။",
+      "Account Inactive": "အကောင့်ပိတ်ထား",
+      "Your account has been deactivated by an admin of your site. Please contact an admin of your site for more information.": "သင့်ဆိုက်အဒ်မင်မှသင်၏အကောင့်ကိုပိတ်ထားလိုက်ပြီ။ ပိုမိုသိရှိလိုပါကသင့်ဆိုက်၏အဒ်မင်ကိုဆက်သွယ်ပါ။",
+      "Get started": "စလိုက်ကြစို့",
+      "Hello": "မင်္ဂလာပါ",
+      "Email": "Email",
+      "Enter [x] (Your [x])": "သင့်-",
+      "Forgot [x]": "-မေ့နေပါသလား",
```

Comment:
What's the plan to account for the `[x]` instances? For example, there is `Enter [x] (Your [x])` and `Enter [x] (to add)`, where they result in different translations

Reply:
I've outlined a solution in the design document [here](https://www.notion.so/calblueprint/Translation-Support-Design-Document-7703018a8c4e403dab8aaa49d2a322e7#6c221ab8002e4ddaad0c1ee82c21dad8). **TLDR**: we'll add an optional argument, `x`, that will nicely handle this case for us!

Reply:
Great, thanks :) sorry should've looked at the design doc first